src/Stratis.Bitcoin.Features.Consensus/CoinViews/InMemoryCoinView.cs

```diff
@@ -98,5 +98,10 @@ public Task<uint256> RewindAsync()
         {
             throw new NotImplementedException();
         }
+
+        public Task<RewindData> GetRewindData(int height)
+        {
+            throw new NotImplementedException();
```

Comment:
Is this expected?

Reply:
Yes. This is not relevant for InMemoryCoinView. The whole coin view concept needs to be redesigned in the future.